curvefs/conf/metaserver.conf

```diff
@@ -6,4 +6,30 @@ metaserver.listen.addr=127.0.0.1:6701
 #
 # metaserver 日志打印目录
 #
-metaserver.common.logDir=/tmp/curvefs/metaserver
\ No newline at end of file
+metaserver.common.logDir=/tmp/curvefs/metaserver
+
+# s3 related
+s3compactwq.enable=False
+s3compactwq.thread_num=2
+s3compactwq.queue_size=5
+s3compactwq.fragment_threshold=20
```

Comment:
add desc for some hard to understand configs？
curve developer and operator can not get the means of these configs.

Reply:
fixed